src/features/courses/course-controller.ts

```diff
@@ -515,19 +555,21 @@ class CourseController {
                         // active: curriculumUnit.active,
                         contentOrder: curriculumUnit.contentOrder,
                         courseId: createdCourse.id,
-                        curriculumUnitId: curriculumUnit.id,
+                        curriculumUnitId: (curriculumUnit instanceof CourseUnitContent) ? curriculumUnit.curriculumUnitId : curriculumUnit.id,
+                        originatingUnitId: (curriculumUnit instanceof CourseUnitContent) ? curriculumUnit.id : undefined,
```

Comment:
I thought sequelize got mad when you specified undefined? unless that is just with where clauses 🤷